recipe_scrapers/finedininglovers.py

```diff
@@ -8,20 +8,34 @@ class FineDiningLovers(AbstractScraper):
     def host(cls):
         return "finedininglovers.com"
 
+    def _is_article(self):
+        """
+        finedinginglovers features recipes and articles. Depending on what is requested the parser needs to handle fields differently.
+        """
+        return 'finedininglovers.com/article/' in self.canonical_url()
```

Comment:
How confident are we that this is an accurate way to distinguish between articles and non-articles, today and longer-term?

(I think it's probably fine.. just wondering if there are ways we could detect any problems with it as soon as possible)

Reply:
good question, i tought about it and decided that currently the importer just fails so it definitely does not make things worse.

One idea that just came to me is that we could entirely scrap the function and just wrap the schema access into a try except block so it will try to access the schema, if that fails it will use the soup and if that fails it just fails. That would account for the case where a recipe schema is present on an article page that does not adhere to the html format thats parsed above. Its also the "python" way of doing things by "asking for forgiveness instead of permission". 

Its an easy fix, what do you prefer ?

Reply:
I don't have a strong opinion -- in fact, I'm genuinely unsure in this case.  Spend a little bit more time thinking about it - I'll leave the decision to you on this one.

Reply:
ok, decided to leave it as is for now, if it ever causes issues we can change it easily